packages/indexer/lib/index.js

```diff
@@ -190,8 +192,8 @@ class CellCollector {
 class TransactionCollector {
   constructor(
     indexer,
-    { lock = null, type = null } = {},
-    { skipMissing = false, includeStatus = true } = {}
+    { lock = null, type = null, fromBlock = null, toBlock = null } = {},
```

Comment:
Please update `index.d.ts` file with the new available parameters. Make sure to document that toBlock is exclusive as well.

Reply:
The `index.d.ts` has been updated.  Besides, I also changed the `toBlock` related logic to  make it  included in the query range, which might cause less confusion for users to read the docs and call the API.